Plugins/Json/include/Acts/Plugins/Json/GeometryJsonKeys.hpp

```diff
@@ -0,0 +1,36 @@
+// This file is part of the Acts project.
+//
+// Copyright (C) 2021 CERN for the benefit of the Acts project
+//
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+#pragma once
+
+#include <string>
+
+namespace Acts {
+
+/// @struct jsonKey
+///
+/// @brief store in a single place the different key used for the material
+/// mapping
+struct jsonKey {
```

Comment:
O, I think we need some naming convention here, I was writing everything in small (no camel case either), because of being an input file, the keys should follow CamelCase though.
```c++
std::string nameKey = "name";
```
That would be my suggestion?

Reply:
Here is the same idea as before I used upper case for the information I wanted to appear at the top when the json object is written. For me when you look at the file the first thing you want to see is the name and ids of the objct that is why I used uppercases

Reply:
What is your opinion on this one @asalzburger ? We cannot use the ordered_json here since it would get converted to normal json by the to_json function. I don't mind having everything be in lower case but I also find it nice to have some useful information at the top of the object in the json file (but that require an uppercase).

Reply:
Ok, we could have it all Upper case, such as "NAME".